### ISSUE TITLE
Don't highlight players once they've just played

### ISSUE THREAD

**Issue by Paul2d** [grifferz/pah-irc#31]
- `<PAH>` **paul** has made their play! We're just waiting on **popey** to make their play.

I know, I just /msg -ed `PAH` and it responded to me personally confirming. I don't need highlights in another window also confirming it. 

Suggested wording: "Another card played, We're just waiting on $player to make their play" or similar?

**Comment by grifferz** [comment]
This will be fixed by the fix to #32.
